app/scripts/early_voting_mgr.js

```diff
@@ -3,7 +3,7 @@ define(
     'jquery', 'moment', 'ejs', 'geojson',
     'map_service',
     'json!vendor/EARLY_VOTING_AddressPoints.geojson',
-    'text!templates/early_voting_sidebar.ejs', 'scrollTo', 'moment_range',
+    'text!templates/early_voting_sidebar.ejs', 'scrollTo', 'moment_range', 'bootstrap', 'bootstrap_min'
```

Comment:
You can replace bootstrap and bootstrap_min with `bootstrapCollapse` - this way we only pull in the component we need.